huaweicloud/services/ces/resource_huaweicloud_ces_alarmrule.go

```diff
@@ -94,8 +97,7 @@ func ResourceAlarmRule() *schema.Resource {
 
 						"dimensions": {
 							Type:     schema.TypeList,
-							Required: true,
-							MaxItems: 3,
```

Comment:
the MaxItemss limitation is not needed now?

Reply:
yes, there is not limit in the API and console